source/tutorials.rst

```diff
@@ -2,6 +2,12 @@
 
 Tutorials
 =========
+.. Note:: 
+	Adam:
+	In general, I find the tutorials to be thorough and easy to read. I feel confident in moving forward with the WEC-Sim applications after the tutorials. There is enought information on how to recreate the tutorials and have them run successfully. However, there’s not a lot of detail on /how/ they run or why they do what they do. Add some explanation on what is going on so that the user can understand and not just replicate. Expand on the differences between the two and what the user should notice between them.
+	I think this section is better placed immediately after the Overview or with the Theory right next to advanced features. The appropriate order largely depends on if someone is a user or developer. User layout might be something like: Overview,  Code Structure, Tutorials, Theory, Advanced Features
+	Developer layout: as is
+
```

Comment:
That's great news! @H0R5E made a similar comment in terms of the order, and I think you should submit that revision via a PR.

What do you think about the tutorials versus the working examples? We don't mention that we *do* provide working examples because we wanted to force users to learn how to use the code... Thoughts?

Reply:
@kmruehl I expanded on the directions for writing the wecSimInputFile.m which I thought needed the most addition. I think that now it feels more guided while still being primarily user drive to go through the information and write it themselves